app/components/ui/TwoColumnContentSection/TwoColumnContentSection.module.scss

```diff
@@ -0,0 +1,111 @@
+@import "../../../styles/utils/_helpers.scss";
+@import "../../../styles/utils/_layoututils.scss";
+
+.twoColumnContentSectionContainer {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-template-rows: 1fr;
+}
+
+.contentContainer_left {
+  grid-column: 2;
+  grid-row: 1;
+  align-self: center;
+}
+
+.imageContainer_left {
+  grid-column: 1;
+  grid-row: 1;
+}
+
+.contentBlock p {
+  color: $text-secondary;
+}
+
+.contentContainer_right .contentBlock {
+  width: 75%;
+  margin-left: 18%;
+  margin-right: 7%;
+}
+
+.contentContainer_left .contentBlock {
+  width: 75%;
+  margin-left: 7%;
+  margin-right: 18%;
+}
+
+.contentContainer_right {
+  grid-column: 1;
+  grid-row: 1;
+  align-self: center;
+}
+
+.imageContainer_right {
+  grid-column: 2;
+  grid-row: 1;
+}
+
+.image {
+  width: 100%;
+}
+
+.contentLinkButton {
+  margin-left: 7%;
+  margin-right: 18%;
+  display: inline-block;
+  margin-top: $general-spacing-md;
+  margin-bottom: $general-spacing-md;
+}
+
+@media screen and (max-width: $break-tablet-p) {
+  .contentBlock h2 {
+    font-size: 1.375rem;
+  }
+
+  .contentBlock p {
+    font-size: 1rem;
+  }
+}
+
+@media screen and (max-width: $break-tablet-s) {
+  .contentContainer_right .contentBlock {
+    width: 100%;
+    margin: 0px 0px;
+    padding: $general-spacing-md $general-spacing-xl;
```

Comment:
The layout of the mockup was confusing but I think this should just be $general-spacing-md on all sides to be vertically in line with the header.

Also nitpick but since right and left have the same css, might be good to group them with a comma rather than declare the css twice.

Reply:
Also on a tablet-ish size screen, the content can butt up against the div above it. Might be good to add some vertical padding to this container.

Reply:
good catch! I chatted with Kyle about the tablet sizes - it doesn't take much content to start messing with the image margin - you can actually see erroneous white space with the proposed content for our415. We could probably do some magic with a clip path or a background image (which I'd prefer not to use - since we can't control what images they use, I'm worried someone will use an image that needs an alt tag), but we also chatted with keeping it simple and increasing the breakpoint for the mobile view. I added that and it fixed this issue as well!